src/content/docs/en/guides/migrate-to-astro/from-create-react-app.mdx

```diff
@@ -361,7 +361,7 @@ If necessary, convert any inline style objects (`style={{ fontWeight: "bold" }}`
 <div style="background-color: #f4f4f4; padding: 1em;">{message}</div>
 ```
 
-Tailwind is supported after installing the [Tailwind integration](/en/guides/integrations-guide/tailwind/). No changes to your existing Tailwind code are required!
+Tailwind is supported after installing the [Tailwind vite plugin](/en/guides/styling/#tailwind). No changes to your existing Tailwind code are required!
```

Comment:
For consistency with other instances in our docs and in Vite docs itself, Vite should be capitalized here.
```suggestion
Tailwind is supported after installing the [Tailwind Vite plugin](/en/guides/styling/#tailwind). No changes to your existing Tailwind code are required!
```